### DISCUSSION TITLE
Federated Reviews

### DISCUSSION THREAD

**Discussion by dcchambers** [bookwyrm-social/bookwyrm#2913] [Ideas]
Hello!

I did search around for this but didn't see anything about it yet. I would love to see "federated reviews" - eg seeing reviews from other Bookwyrm instances for a given book. I'm not even sure if it's technically possible to be honest. I think it would go a long way towards helping the social aspects of the decentralized platform reach that critical mass required for sustained success. Ideally it's something that users can toggle on/off - so those that are not interested in reviews outside of their instance can ignore other instances in the network.

One of the ways I heavily used Goodreads (and hopefully Bookwyrm) was reading through reviews on a book I am interested in. The decentralized nature of Bookwyrm makes that hard to do as reviews are spread out on different instances.

Cheers ✌️

**Comment by mouse-reeve** [comment]
Hi, thanks for reaching out! BookWyrm does show federated reviews, but the
way it finds reviews is a little convoluted. There are a couple changes I
have planned to make it work better, like loading a remote review by its
url, and searching remote instances for reviews of a specific book, but I
just haven't had time to do the work yet.

**Reply by dato** [reply]
I just saw this... would "loading a remote review by its url" apply to status updates too, as I was asking about in #2990? Is it the same work that is needed?
